src/app/learn/get-started/advanced/add-entry-components/index.html

```diff
@@ -1,32 +1,32 @@
 <stache
-  pageTitle="Add Entry Components"
+  pageTitle="Entry Components"
```

Comment:
Would you want to also rename this folder to `entry-components` and add a redirect from `add-entry-components` to `entry-components`?

Reply:
I renamed the folder and added a redirect.